.yarnrc.yml

```diff
@@ -0,0 +1,27 @@
+checksumBehavior: ignore
+
+defaultSemverRangePrefix: ""
+
+enableTelemetry: false
```

Comment:
i personally think its always good to enable telemetry since it helps the devs who build the tool but just my opinion :)

Reply:
if the devs were evil and were collecting private data from us they wouldn't have done it through the opt-outable telemetry option